model/project_ref_test.go

```diff
@@ -2043,6 +2040,14 @@ func TestCreateNewRepoRef(t *testing.T) {
 	assert.False(t, repoRef.TaskSync.IsPatchEnabled())
 	assert.True(t, repoRef.ParameterStoreVarsSynced)
 
+	assert.NotContains(t, repoRef.Admins, "bob")
+	assert.NotContains(t, repoRef.Admins, "other bob")
+	assert.Contains(t, repoRef.Admins, "me")
+	users, err := user.FindByRole(GetRepoAdminRole(repoRef.Id))
+	assert.NoError(t, err)
+	require.Len(t, users, 1)
+	assert.Equal(t, users[0].Id, "me")
```

Comment:
nit: I know the linter is gonna eventually catch this, but the expected value for equals assertion is the second parameter: `assert.Equal(t, "me", users[0].Id)`